custom-metrics-stackdriver-adapter/pkg/adapter/translator/utils/filter_builder_test.go

```diff
@@ -245,6 +245,52 @@ func TestFilterBuilder_WithPods_Multiple_legacy(t *testing.T) {
 	expectFilterBuilder(actual).toEqual(expected).report(t)
 }
 
```

Comment:
Please add test cases for node metric labels.

Reply:
as discussed, we are only using `metric.labels.node` now, so it's covered by other tests